lib/src/channels/http.dart

```diff
@@ -110,7 +117,7 @@ class LoggingHttpClient implements HttpClient {
 
   @override
   void close({bool force = false}) {
-    _todo('close');
+    _log.log('CLOSE${force ? " • (forced)" : ""}');
```

Comment:
I don't know if the `forced` is valuable to users. If we're printing `close` in the log, we should also saw what opened. From the HttpClient docs:

```
The HttpClient supports persistent connections and caches network connections to reuse them for multiple requests whenever possible. This means that network connections can be kept open for some time after a request has completed. Use HttpClient.close to force the HttpClient object to shut down and to close the idle network connections.
```

So the HttpClient could have 0 to n open network connections. Close might be useful to print if we could say when we open and close an underlying network connection (and not just a GET or POST request, several of which could run on a connection).

I guess I'd lean towards not logging anything for now, until as have a better way to measure what the underlying impl is doing.

Reply:
I was thinking the same thing.  Will remove.